mitmproxy/events.py

```diff
@@ -0,0 +1,85 @@
+import re
+from dataclasses import dataclass, is_dataclass, fields
+from typing import ClassVar, Any, Dict, Type, Set, List, TYPE_CHECKING, Sequence
+
+import mitmproxy.flow
+
+if TYPE_CHECKING:
+    import mitmproxy.addonmanager
+    import mitmproxy.log
+
+
+class MitmproxyEvent:
```

Comment:
nitpick: personally I always try to avoid writing `mitmproxy` with a capital letter. It looks odd and somehow doesn't fit. 

Also, what purpose serves adding "Mitmproxy..." here? The full import path is now dual redundant:
`mitmproxy.events.MitmproxyEvent`

https://xkcd.com/244/

Reply:
Picked this to have a distinct class name from `mitmproxy.proxy.events.Event`. Agreed on everything else, let's unify the discusson above.